src/App.js

```diff
@@ -1,17 +1,197 @@
-import React, { Component } from 'react';
-import Letters from './components/Letters';
-import './App.css';
-import Score from './components/Score';
-
-class App extends Component {
-  render() {
-    return (
-      <div>
-        <Score />
-        <Letters />
-      </div>
-    );
+import React, { Component } from "react";
+import Letters from "./components/Letters";
+import Score from "./components/Score";
+import Solution from "./components/Solution";
+import "./App.css";
+// import Background from '../public/insideout.jpg';
+let words =  [
+  {word:"SATISFY",hint:"Awesome mood",secondHint:"Very good mood"},
+  {word:"HAPPINESS",hint:"Yellow mood",secondHint:"Look at the background"},
+  {word:"FEAR",hint:"Purple mood",secondHint:"Look at the background"},
+  {word:"ANGER",hint:"Red mood",secondHint:"Look at the background"},
+  {word:"DISGUSTE",hint:"Green mood",secondHint:"Look at the background"},
+  {word:"SADNESS",hint:"Blue mood",secondHint:"Look at the background"}];
+  let rand = Math.floor((Math.random() * words.length));
+  let toShow = false; //hide
```

Comment:
avoid global variables
certainly, data that affects the UI should be in `state`

Reply:
if you're simulating a DB, use `componentDidMount`